frontend/src/modelTypes/ITaskDuplicate.ts

```diff
@@ -0,0 +1,8 @@
+import type {IAbstract} from './IAbstract'
+import type {ITask} from './ITask'
+
+export interface ITaskDuplicate extends IAbstract {
+	taskId: number
+	projectId: number
+	duplicatedTask: ITask | null
+}
```

Comment:
@kolaente: First I wanted to note that the final new line is missing in the three new ts files, but then I realised that we have `insert_final_newline = false` in the `.editorconfig` file. Without final empty line GitHub displays it like this:

<img width="475" alt="Screenshot 2025-05-23 at 11 04 06" src="https://github.com/user-attachments/assets/2b978c0d-ccae-4e66-b123-1a3d54e5e9e1" />

Reply:
uhhh that should not be the case, I've changed it in dc6ef8b4be000b3dfe50c0e71e5009d764d31ef6